llvm/lib/Target/AIE/AIE2AddrSpace.h

```diff
@@ -39,6 +39,7 @@ enum class AddressSpaces {
 };
 
 enum class DMBanks { A, B, C, D };
+enum class TileMemory { TileMemory = 4 };
```

Comment:
Nit: I think there should be a single enum, e.g. `enum class AIEBanks { A, B, C, D, Tile };`

Reply:
Update DMBanks/TileMemory to AIEBanks